### PR TITLE
[INLONG-820][Doc] Add deployment os require

### LINKED ISSUE #820: [Improve] Add deployment os require and tubemq as database config statement
### Description

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by liaosunny123**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #820 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: docs/deployment/standalone.md [hunk 1]
```diff
@@ -36,7 +37,7 @@ In `conf/inlong.conf`, configure the parameters according to the actual situatio
 ```shell
 # local IP
 local_ip=
-# message queue: pulsar or kafka
+# message queue: pulsar or kafka or tubemq
```
Comment (dockerzhang) [inline docs/deployment/standalone.md:40]: Using pulsar or kafka is more easy for `standalone mode` , so tubemq is no a recomanded.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/deployment/standalone.md [hunk 2]
```diff
@@ -36,7 +37,7 @@ inlong-tubemq-manager/lib/
 ```shell
 # 本地机器 IP
 local_ip=
-# 消息队列服务： pulsar 或者 kafka
+# 消息队列服务： pulsar 或者 kafka 或者 tubemq
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/deployment/standalone.md:40]: ditto

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/deployment/standalone.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ sidebar_position: 1
 - MySQL 5.7+
 - [Flink 1.13.5](https://nightlies.apache.org/flink/flink-docs-release-1.13/docs/try-flink/local_installation/)
 - [Docker](https://docs.docker.com/engine/install/) 19.03.1+
+- Linux或Mac环境
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/deployment/standalone.md:9]: Linux或Mac环境
->
Linux 或 Mac 环境

**Comment by liaosunny123** [comment]
I also find that there may be a typo, I find the database config can only support Mysql according to `inlongctl`

**Review by dockerzhang — APPROVED** [review]

**Comment by dockerzhang** [comment]
> I also find that there may be a typo, I find the database config can only support Mysql according to `inlongctl`. Should I push here or create a new issue?

that would be very welcome.

**Review by vernedeng — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Pull request merged**